ostelco-core/Network/PrimeAPI.swift

```diff
@@ -144,11 +139,16 @@ open class PrimeAPI: BasicNetwork {
     }
 
     /// - Returns: A Promise which when fulfilled will contain the Stripe Ephemeral Key
-    public func stripeEphemeralKey(stripeAPIVersion: String) -> Promise<[String: AnyObject]?> {
+    public func stripeEphemeralKey(with request: StripeEphemeralKeyRequest) -> Promise<[String: AnyHashable]> {
```

Comment:
Stripe expects the key as `[AnyHashable : Any]?`. hope this change will still work 
https://stripe.dev/stripe-ios/docs/Type%20Definitions.html#/c:STPBlocks.h@T@STPJSONResponseCompletionBlock

Reply:
`String` conforms to `AnyHashable` and `AnyHashable` definitely conforms to `Any`. However, since it's clear that's the type they want, I'll update it. 

While you can pass in a nullable object because this is obj-c code, I can't imagine any scenario in which passing `nil` would lead to success there, which is why I'm having this throw if it doesn't get anything back.